internal/publish/database/exposure.go

```diff
@@ -155,19 +159,31 @@ func generateExposureQuery(criteria IterateExposuresCriteria) (string, []interfa
 		q += fmt.Sprintf(" AND NOT (regions && $%d)", len(args)) // Operation "&&" means "array overlaps / intersects"
 	}
 
+	if criteria.RevisedKeys {
+		q += " AND revised_at IS NOT NULL"
+	}
+
 	// It is important for StartTimestamp to be inclusive (as opposed to exclusive). When the exposure keys are
 	// published, they are truncated to a time boundary (e.g., time.Hour). Even though the exposure keys might arrive
 	// during a current open export batch window, the exposure keys are truncated to the start of that window,
 	// which would make them fall into the _previous_ (already processed) batch if StartTimestamp is exclusive
 	// (in the case where the publish window and the export period align).
 	if !criteria.SinceTimestamp.IsZero() {
 		args = append(args, criteria.SinceTimestamp)
-		q += fmt.Sprintf(" AND created_at >= $%d", len(args))
+		if criteria.RevisedKeys {
```

Comment:
Is there a better way to avoid this branching logic? I see it a few times...

Reply:
done